pywinauto/unittests/test_uiawrapper.py

```diff
@@ -2115,6 +2115,7 @@ def tearDown(self):
         def test_issue_443(self):
             """Test .set_focus() for window that is not keyboard focusable"""
             self.dlg.minimize()
+            time.sleep(0.2)
```

Comment:
Maybe it's worth using `wait_until_passes(...)` for `.is_minimized()` result.

Reply:
Done by `wait_until`